src/main/java/at/ac/fhcampuswien/fhmdb/controller/AbstractViewController.java

```diff
@@ -42,7 +42,7 @@ public abstract class AbstractViewController {
     protected SVGPath cross = new SVGPath();
     protected SVGPath burger = new SVGPath();
 
-    protected final ObservableList<Movie> movies = FXCollections.observableArrayList();
+    public final ObservableList<Movie> movies = FXCollections.observableArrayList();
```

Comment:
rather than making this public, I would add a getter for it

Reply:
This field is final, so we can't use setter here. Should I make it just protected with getter & setter?

Reply:
I didn't say we need a setter, I was just referring to the getter